### DISCUSSION TITLE
Having Issues with Connecting APIs to my localhost server

### DISCUSSION THREAD

**Discussion by MaiAditya** [airbytehq/airbyte#44303] [Connector Questions]
AttributeError: module 'airbyte_api.models' has no attribute 'SchemeClientCredentials'
Getting this error when trying to authenticate using this code 
import airbyte_api
from airbyte_api.models import SourceCreateRequest, SourceGoogleSheets

# Initialize the Airbyte API
s = airbyte_api.AirbyteAPI(
    base_url="http://localhost:8000",  # replace with the URL of your local server
    security=airbyte_api.models.Security(
        client_credentials=airbyte_api.models.SchemeClientCredentials(
            client_id="",
            client_secret="",
            TOKEN_URL="",
        ),
    ),
)

Anyone knows how to resolve the connection issue with the localhost server please help!

**Comment by Abdi-Arslene** [comment]
I'm facing the same issue is there a fix ?

**Comment by Abdi-Arslene** [comment]
@MaiAditya Did you find a solution for this ? i'm getting the same issue there.

[a comment by an automated account is not shown]
